cvrf2csaf/cvrf2csaf.py

```diff
@@ -34,6 +34,9 @@ class DocumentHandler:
     5. Combining it to the final JSON and writing the result to a file
     """
 
+    TOLERATED_ERRORS_SUBSTR = ["}ScoreSetV3': This element is not expected. Expected is one of ( {http://docs.oasis-open.org/csaf/ns/csaf-cvrf/v1.2/vuln}ScoreSetV2, {http://docs.oasis-open.org/csaf/ns/csaf-cvrf/v1.2/vuln}ScoreSetV3 ).",
+                        "ScoreSetV3': This element is not expected. Expected is ( {http://docs.oasis-open.org/csaf/ns/csaf-cvrf/v1.2/vuln}ScoreSetV3 )."]
```

Comment:
Nit: `"ScoreSetV3':` &rightarrow; `"ScoreSetV3:`

Reply:
@zpevma In which case does that error message occur?

Reply:
for example this:
```
<CVSSScoreSets>
      <ScoreSetV2>
        <BaseScoreV2>7.5</BaseScoreV2>
        <VectorV2>AV:A/AC:M/Au:N/C:N/I:C/A:P</VectorV2>
      </ScoreSetV2>
      <ScoreSetV3>
        <BaseScoreV3>7.5</BaseScoreV3>
        <VectorV3>CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H</VectorV3>
      </ScoreSetV3>
      <ScoreSetV3 xmlns="https://www.first.org/cvss/cvss-v3.0.xsd">
        <BaseScoreV3>7.5</BaseScoreV3>
        <VectorV3>CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H</VectorV3>
      </ScoreSetV3>
</CVSSScoreSets>
```
`ScoreSetV3` element with namespace appears after `ScoreSetV3` without namespace. Then the error message is different from previous one.